.scripts/build_steps.sh

```diff
@@ -5,6 +5,8 @@
 # changes to this script, consider a proposal to conda-smithy so that other feedstocks can also
 # benefit from the improvement.
 
+git config --global --add safe.directory '*'
```

Comment:
I can revert this, but it also wouldn't hurt I think, and it fixes execution with podman instead of docker for me.

Reply:
Let's revert when we are done here